tests/ocs-monkey/test_ocs_monkey.py

```diff
@@ -15,46 +15,90 @@
 def test_ocs_monkey():
     ocs_monkety_dir = "/tmp/ocs-monkey"
     # ocs-monkey run time in seconds
-    run_time = 3200
+    run_time = 3600
```

Comment:
thinking maybe to have the run_time a config value that we will be able to pass to run-ci